### DISCUSSION TITLE
ビル経営ゲームに関する質問

### DISCUSSION THREAD

**Discussion by toakiryu** [selcold/scratch-building#2] [Q&A]
# 気になることなどをここで聞いて解決しよう

**Comment by teln-banana** [comment]
青い猫と普通の猫の違いは何ですか？

**Reply by Aikame-Game** [reply]
見てみた感じ通常の猫より歩くスピードが速い猫です。発見したのはこのくらいです。もしかしたら沢山買ってくれるというのもあるかもしれません。

**Comment by discovery1104** [comment]
以下のエラーが出ました
[Error: ENOENT: no such file or directory, open 'D:\html\scratchbuilding\scratch-building\.next\BUILD_ID'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'D:\\html\\scratchbuilding\\scratch-building\\.next\\BUILD_ID'
}
助けtえください
npm i したあとにnpm startしたんですけどだめでしたかね

**Reply by toakiryu** [reply]
コマンドは`num start`ではなく`npm rum dev`です。そして、`\html\scratchbuilding\scratch-building\`から起動するより、エディターで`scratch-building`フォルダーを開いて作業をした方がいいと思います。

**Reply by discovery1104** [reply]
ありがとござます

**Comment by Aikame-Game** [comment]
広告はどうやって載せるのですか？

**Reply by toakiryu** [reply]
[スタジオ](https://scratch.mit.edu/studios/34675120/)にプロジェクトを追加してください

**Reply by Aikame-Game** [reply]
すぐに反映される！？すげぇーなぁー

**Comment by saijaku1** [comment]
?

**Reply by toakiryu** [reply]
?

**Comment by kamepengin** [comment]
ショップはいつ復活しますか?

**Reply by Aikame-Game** [reply]
現在 https://ssm.scra.cc というサイトを見るとクラウド変数が止まっていることが確認できますが、サイトなどの同期のためクラウド変数が機能しないとショップが機能しない仕組みになっています。スクラッチ運営側がいつ復活するかによります。
